spam/03_spam_data_slicing_tutorial.py

```diff
@@ -37,31 +37,30 @@
 
 # %% [markdown]
 # _Note:_ this tutorial differs from the labeling tutorial in that we use ground truth labels in the train split for demo purposes.
-# SFs are intended to be used *after the training set has already been labeled* by LFs (or by hand) in the trainind data pipeline.
+# SFs are intended to be used *after the training set has already been labeled* by LFs (or by hand) in the training data pipeline.
 
 # %%
 from utils import load_spam_dataset
 
-df_train, df_valid, df_test = load_spam_dataset(
-    load_train_labels=True, include_dev=False
-)
+df_train, df_valid, df_test = load_spam_dataset(load_train_labels=True, split_dev=False)
 
 df_train.head()
 
 
 # %% [markdown]
 # ## 1. Write slicing functions
 #
-# We leverage *slicing functions* (SFs) — an abstraction that shares syntax with *labeling functions*, which you should already be familiar with.
-# If not, please see the [intro tutorial](https://github.com/snorkel-team/snorkel-tutorials/blob/master/spam/01_spam_tutorial.ipynb).
+# We leverage *slicing functions* (SFs) — an abstraction that shares syntax with *labeling functions*, which you should already be familiar with (if not, please see the [intro tutorial](https://github.com/snorkel-team/snorkel-tutorials/blob/master/spam/01_spam_tutorial.ipynb)).
+#
 # A key difference: whereas labeling functions output labels, slicing functions output binary _masks_ indicating whether an example is in the slice or not.
+# Each slice represents some noisily-defined subset of the data (corresponding to an SF) that we'd like to programmatically monitor.
 
 # %% [markdown]
 # In the following cells, we use the [`@slicing_function()`](https://snorkel.readthedocs.io/en/redux/packages/_autosummary/slicing/snorkel.slicing.slicing_function.html#snorkel.slicing.slicing_function) decorator to initialize an SF that identifies shortened links the spam dataset.
 # These links could redirect us to potentially dangerous websites, and we don't want our users to click them!
 # To select the subset of shortened links in our dataset, we write a regex that checks for the commonly-used `.ly` extension.
 #
-# You'll notice that the slicing function is noisily defined — it doesn't represent the ground truth for all short links.
+# You'll notice that the `short_link` SF is noisy — it doesn't represent the ground truth for all short links.
```

Comment:
"represent the ground truth" -> can we just say it's only accurate some of the time?

Reply:
Updated:

> it might not be accurate for _all_ examples